code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -248,6 +250,8 @@
 		if(G.vision_correction)
 			if(HAS_TRAIT(src, TRAIT_NEARSIGHT))
 				overlay_fullscreen("nearsighted", /atom/movable/screen/fullscreen/impaired, 1)
+			if(HAS_TRAIT(src, TRAIT_FARSIGHT) && !HAS_TRAIT(src, TRAIT_ILLITERATE)) //ORBSTATION: Farsighted quirk handling
```

Comment:
Similarly I don't think you need to check `HAS_TRAIT` before adding it either, which might cause similar bugs.